### PR TITLE
Add files from task HTML&CSS Practice

### PR DESCRIPTION
**Pull request by deveLabR**
### HTML & CSS Practice

[Demo](https://develabr.github.io/hooli-style-popup/) |
[Code base](https://github.com/deveLabR/hooli-style-popup)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

**Comment by deveLabR** [comment]
I have finished the self-checks, and I want to believe that it does not sound too presumptuous

File: submissions/develabr/hooli-style-popup/index.html [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+          <li><a class="nav__menu__link" href="#">Mail</a></li>
+          <li><a class="nav__menu__link" href="#">Images</a></li>
+          <li>
+            <input
+              id="menu__checkbox"
+              class="menu__checkbox__input"
+              type="checkbox"
+            />
+            <label for="menu__checkbox" class="menu__checkbox__label">
+              <img src="./images/popup-button.png" alt="popup-button" />
```
Comment (A-Ostrovnyy) [inline submissions/develabr/hooli-style-popup/index.html:23]: Read how and when to populate alt attribute value [Как правильно написать alt-текст](https://htmlacademy.ru/blog/articles/alt-text)

File: submissions/develabr/hooli-style-popup/index.html [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+            </label>
+            <div class="menu__popup">
```
Comment (A-Ostrovnyy) [inline submissions/develabr/hooli-style-popup/index.html:25]: Fix mobile version
image

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good work!

File: submissions/develabr/hooli-style-popup/index.html [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+              <img src="./images/popup-button.png" alt="Popup button" />
+            </label>
+            <div class="menu__popup">
+              <ul class="menu__popup__list">
+                <li class="menu__popup__item">
+                  <a class="menu__popup__link" href="#"
+                    ><img
+                      class="menu__popup__image"
+                      src="./images/calendar.png"
+                      alt="Calendar icon"
```
Comment (A-Ostrovnyy) [inline submissions/develabr/hooli-style-popup/index.html:32]: image

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
One last issue)

**Review by A-Ostrovnyy — APPROVED** [review]
Awesome!

**Pull request merged**